### ISSUE TITLE
need a mechanism to indicate if something is readonly or readwrite at run time

### ISSUE THREAD

**Issue by WAvdBeek** [one-data-model/language#21]
issue with temperature:
it can indicate the current temperature (read) or indicate the set point (read write)
this can be distinguished on run time.

**Comment by mjkoster** [comment]
Yes, readability and writeability are often part of instance bindings. Do they really need to be part of the model for interoperability?

**Comment by gerickson** [comment]
I think these will need to be part of the high-level meta model, even if only for lossless use in existing ecosystems.
